glossary.html

```diff
@@ -2,6 +2,14 @@
 <html>
 <head>
     <title>Shivani Sharma,s CSD120 Website</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+	<link rel="stylesheet" href="styles/main.css">
+
+	<link rel="icon" href="images/favicon.ico" type="image/x-icon" />
+
+
+	<link href="https://fonts.googleapis.com/css?family=Lora|Open+Sans&display=swap" rel="stylesheet">
```

Comment:
Missing Google fonts link.  (I just picked two arbitrary fonts on Google Fonts and applied them for you here.)